box2dworld.cpp

```diff
@@ -137,6 +139,8 @@ Box2DWorld::Box2DWorld(QObject *parent) :
 
 {
     mWorld.SetDestructionListener(this);
+    if(!defaultWorld)
```

Comment:
Please remember that we are putting a space between `if` and `(` (also in `Box2DBody::componentComplete`).